src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -3413,6 +3413,34 @@ namespace CondenserLoopTowers {
             OutputReportPredefined::PreDefTableEntry(
                 state, state.dataOutRptPredefined->pdchMechType, this->Name, DataPlant::PlantEquipTypeNames[static_cast<int>(this->TowerType)]);
             OutputReportPredefined::PreDefTableEntry(state, state.dataOutRptPredefined->pdchMechNomCap, this->Name, this->TowerNominalCapacity);
+
+            // create std 229 new table for cooling towers and fluid coolers
+            OutputReportPredefined::PreDefTableEntry(
+                state, state.dataOutRptPredefined->pdchCTFCType, this->Name, DataPlant::PlantEquipTypeNames[static_cast<int>(this->TowerType)]);
+            OutputReportPredefined::PreDefTableEntry(state,
+                                                     state.dataOutRptPredefined->pdchCTFCCondLoopName,
+                                                     this->Name,
+                                                     this->plantLoc.loopNum > 0 ? state.dataPlnt->PlantLoop(this->plantLoc.loopNum).Name : "N/A");
```

Comment:
This protection is fine, but is it necessary? When would a fluid cooler or tower not be connected to a plant loop? That seems possible only as an error condition. Or should "N/A" be "Not found"?

Reply:
Yes, this is a very generic protection---actually I remember I encountered many crashes in CI test files (or unit tests which might not be complete cases) with similar "plantloop cannot find smoothing and then crash" cases (not necessarily this type of cond loop for fluid coolers though). I cannot speak for sure in this specific scenario if this is going to be always good without the check. For the text, whatever suitable would be fine---I think for the other similar scenarios "N/A" was used---or maybe I can change the other places to "Not found".